modules/apps/site-initializer/site-initializer-cms/src/main/resources/site-initializer/fragments/group/sidebar/index.css

```diff
@@ -16,6 +16,7 @@
 		border-right: 1px solid #9b92a5;
 		display: flex;
 		grid-area: sidebar;
+		margin-top: 56px;
```

Comment:
Instead of this we should use `top: var(--control-menu-height);` on `navigation-bar`. The `navigation-bar` is also missing the variant class `navigation-bar-light`.